### ISSUE TITLE
TypeHintDeclaration with disabled enableNullableTypeHints does not work with optional parameters

### ISSUE THREAD

**Issue by VasekPurchart** [slevomat/coding-standard#128]
The fixer for `enableNullableTypeHints` disabled tries to add `= null` to the parameter even if it is already there.

Minimal reproducible example:

```php
<?php

class Foo
{

	/**
	 * @param string|null $baz
	 */
	public function bar($baz = null)
	{
		
	}

}
```

Is converted to:

```php
<?php

class Foo
{

	/**
	 * @param string|null $baz
	 */
	public function bar(string $baz = null = null)
	{
		
	}

}
```

So it produces `string $baz = null = null`, which is obviously incorrect. I think it is needed to look if the null (or probably any other default value) is already present.

**Comment by kukulich** [comment]
Fixed in e083ac04fb7a0d22db7fbe3a04c99db52a71a4ae.

**Comment by VasekPurchart** [comment]
Thanks :)
